src/extension.ts

```diff
@@ -8,10 +8,52 @@ import * as vscode from 'vscode';
 // this method is called when your extension is activated
 // your extension is activated the very first time the command is executed
 export async function activate(context: vscode.ExtensionContext) {
+    console.log("activated!")
+    // Add status bar button
+    let statusBarJsim = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 100);
```

Comment:
I believe it would be good to have the buttons there as it'll be more intuitive for most users. How about adding an option in the extension settings to allow (power) users to turn off these status bar buttons? As of now commands can already be used: `compstruct-vscode.jsimTerminal` and `compstruct-vscode.bsimTerminal`.

Reply:
Implemented settings to toggle status bar buttons

Reply:
Hmm alright, since the extension is only activated when the user opens `.jsim`, `.uasm` or `.tmsim` files, I think this is an acceptable tradeoff. The settings to toggle the status bar buttons are also a nice touch as they provide a nice option for users who would like to opt-out. Thanks!